linux_os/guide/services/ntp/service_ntp_enabled/rule.yml

```diff
@@ -50,9 +50,14 @@ template:
     vars:
         servicename: ntp
 
-{{% if prodtype in ["rhel8", "rhel9", "sle15"] %}}
+{{% if prodtype in ["rhel7", "rhel8", "rhel9", "sle15"] %}}
 warnings:
     - general:
+        {{% if prodtype == "rhel7" %}}
```

Comment:
Maybe it would be better if the warning message would recommend to use the `service_ntpd_enabled` rule instead.

Reply:
Good idea, I added it in 6e60138a213c746676da1df6c74832e7771abb0e ..